app/views/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for(@user, layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-6") do |f|
+        %header
+          %h1 Register
+        %fieldset
+          = f.email_field :email, label: "Email Address"
+          = f.password_field :password
+          = f.text_field :full_name, label: "Full Name"
+        %fieldset.actions.control-group.col-sm-offset-2
+          .controls
+            %input(type="submit" value="Sign Up" class="btn btn-default")
```

Comment:
No reason, that's just how it was in the solution video. I'm still getting used to haml syntax so it didn't jump out at me when I was doing it. It does make more sense now that it is fixed though. Thanks for picking this up!